scixhelp/index.md

```diff
@@ -4,17 +4,1271 @@ title: 'SciX Help'
+      </ul>
+    </div>
+  </div>
+  
+  <div class="faq-item" id="export-citations">
+    <div class="faq-question" onclick="toggleFAQ('faq14')">
+      <h3>Can I export my search results to a citation manager like EndNote or Zotero?</h3>
+      <span class="faq-toggle">+</span>
+    </div>
+    <div class="faq-answer" id="faq14">
+      <p>Yes! Export your search results to citation managers:</p>
+      <ul>
+        <li><strong>Bulk Actions menu:</strong> Export search results → most scientific reference formats</li>
+        <li><strong>Supported formats:</strong> EndNote, Zotero, BibTeX, RIS, and more</li>
+        <li><strong>Coverage:</strong> Earth and space sciences, physics, and mathematics</li>
+        <li><strong>Libraries too:</strong> Same export options available for saved libraries</li>
+      </ul>
+      <p>Learn more about <a href="{{ site.baseurl }}/scixhelp/actions-scix">export options</a> and <a href="{{ site.baseurl }}/scixhelp/libraries-scix">library management</a>.</p>
+    </div>
+  </div>
+  
+  <div class="faq-item" id="search-history">
+    <div class="faq-question" onclick="toggleFAQ('faq15')">
+      <h3>Is my search history saved? Can I retrieve my searches later?</h3>
+      <span class="faq-toggle">+</span>
+    </div>
+    <div class="faq-answer" id="faq15">
+      <p>For privacy reasons, we don't track or save your search history:</p>
+      <ul>
+        <li><strong>No personal tracking:</strong> We collect aggregate search data but can't track back to you</li>
+        <li><strong>Email alerts:</strong> Use bell button → set up notifications for your search</li>
+        <li><strong>Save searches:</strong> Create account → <strong>Bulk Actions</strong> → export to Library → note query in description</li>
+        <li><strong>Revisit anytime:</strong> Access saved searches through your libraries</li>
+      </ul>
+      <p>Learn more about <a href="{{ site.baseurl }}/scixhelp/libraries-scix">library management</a> and <a href="{{ site.baseurl }}/scixhelp/userpreferences-scix">user preferences</a>.</p>
+    </div>
+  </div>
+  
+  <div class="faq-item" id="discipline-selection">
+    <div class="faq-question" onclick="toggleFAQ('faq16')">
+      <h3>How and why would I specify my discipline?</h3>
+      <span class="faq-toggle">+</span>
+    </div>
+    <div class="faq-answer" id="faq16">
+      <p>Specify your discipline to boost relevant content:</p>
+      <ul>
+        <li><strong>Top-left menu:</strong> Select preferred discipline → boosts relevant content to top</li>
+        <li><strong>General Science:</strong> Primarily physics, computer science, and chemistry content</li>
+        <li><strong>Left panel:</strong> Select collection by discipline → filters out other disciplines</li>
+        <li><strong>Account setting:</strong> Create account → discipline stays selected for future searches</li>
```

Comment:
`<strong>Update your account settings:</strong> Update your <a href="https://scixplorer.org/user/settings">default collection</a> to automatically filter by collection in the future.`